### ISSUE TITLE
We using this repo for anything?

### ISSUE THREAD

**Issue by trusktr** [docsifyjs/art#1]
Is the Docsify logo made in [Sketch](https://www.sketch.com/) (I haven't checked what's inside because Sketch is only for macOS, I'm on Linux mostly).

**Comment by trusktr** [comment]
@anikethsaha

**Comment by anikethsaha** [comment]
I dont know about this repo. Never noticed this 😄

**Comment by Koooooo-7** [comment]
It seems to store our icon original file.

**Comment by trusktr** [comment]
I don't think we really need this repo. We already have the icon SVG elsewhere. I vote to delete it. Wdyt?
